src/Actions/Actions/LoadAction.cs

```diff
@@ -53,9 +57,22 @@ internal static LoadActionParts LoadSnapshotZip(string path)
                 SnapshotMetaInfo meta = SnapshotMetaInfo.DeserializeFromStream(metadataPart);
                 metadataPart.Close();
 
+                IReadOnlyDictionary<int, CustomProperty> CustomProperties;
+                try
+                {
+                    var customPropertiesPart = (from p in parts where p.Uri.OriginalString == "/" + SaveAction.customPropsFileName select p.GetStream()).First();
+                    CustomProperties = LoadCustomProperties(customPropertiesPart);
+                    customPropertiesPart.Close();
+                }
+                catch (InvalidOperationException e)  // Gets thrown if custom properties map doesn't exist in file
```

Comment:
This catches the "no stream exists" case, but should we also consider cases where the stream exists but is somehow broken? Maybe 2 catch blocks--one that silently ignores the expected case, and one that reports the unexpected case